relay-kafka/src/config.rs

```diff
@@ -344,11 +355,11 @@ mod tests {
     #[test]
     fn test_kafka_config() {
         let yaml = r#"
-events: "ingest-events-kafka-topic"
+ingest-events: "ingest-events-kafka-topic"
```

Comment:
Can we have another test that verifies the old names still work (otherwise we break self hosted)

Reply:
Sounds good, added the test